connector/src/test/java/it/eng/connector/integration/negotiation/ContractNegotiationFinalizeIntegrationTest.java

```diff
@@ -120,27 +118,18 @@ public void handleFinalizeEventTest() throws Exception {
     	TimeUnit.SECONDS.sleep(1);
     	
     	//check if Transfer Process is initialized
-    	final ResultActions tp =
-				mockMvc.perform(
-						get(ApiEndpoints.TRANSFER_DATATRANSFER_V1).param("role", IConstants.ROLE_CONSUMER)
-						.with(user(TestUtil.CONNECTOR_USER).password("password").roles("ADMIN"))
-						.contentType(MediaType.APPLICATION_JSON));
-		
-		tp.andExpect(status().isOk())
-			.andExpect(content().contentType(MediaType.APPLICATION_JSON));
-		
-		JsonNode jsonNode = jsonMapper.readTree(tp.andReturn().getResponse().getContentAsString());
-		TransferProcess transferProcess = TransferSerializer.deserializePlain(jsonNode.findValues("data").get(0).get(jsonNode.findValues("data").get(0).size()-1).toString(), TransferProcess.class);
+		TransferProcess transferProcess = transferProcessRepository.findByAgreementId(agreement.getId()).get();
 		
 		assertEquals(TransferState.INITIALIZED, transferProcess.getState());
-		assertNotNull(transferProcess.getAgreementId());
 		assertNotNull(transferProcess.getCallbackAddress());
 		assertNotNull(transferProcess.getRole());
 		assertNotNull(transferProcess.getDatasetId());
+		assertEquals("temporary_provider_pid", transferProcess.getProviderPid());
```

Comment:
Maybe we could extract those string to some constants/fields?
temporary_provider_pid
temporary_consumer_pid